### PR TITLE
two questions answered

### PR DESCRIPTION
**Pull request by CrowdHailer**

### CODE REVIEW

File: faq/index.md [hunk 1]
```diff
@@ -16,37 +16,57 @@ You can always fall back to using email authentication to login. If your device
...
-If you forget to remove all your data from the device when you sell it you can remove authorisation for that device inside your DID account at any time (this feature is currently in development).
+If you forget to remove all your data from the device when you sell it you can remove authorisation for that device inside your DID.app account at any time (this feature is currently in development).
 
 <a name="how-is-authentication-via-email-secured"></a>
 ### How is account recovery via email secured?
 
 A link containing a single use code is generated and emailed to the user.
 As only the owner of the email address can receive the link only they can access the account.
 
+### Where are the private keys for my device stored.
```
Comment (richardesigns) [inline faq/index.md:34]:
```suggestion
### Where are the private keys for my device stored?
```

File: faq/index.md [hunk 2]
```diff
@@ -16,37 +16,57 @@ You can always fall back to using email authentication to login. If your device
...
+
+Key pairs are generated and stored by the browser.
```
Comment (richardesigns) [inline faq/index.md:36]:
```suggestion
Key pairs are generated and stored by your browser.
```

File: faq/index.md [hunk 3]
```diff
@@ -16,37 +16,57 @@ You can always fall back to using email authentication to login. If your device
...
+
+From within the browser it is not possible to access the private key, only a reference to a private key.
+Private key references are stored in [indexeddb](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API/Using_IndexedDB),
+To see all the key references stored select the storage tab in web developer tools.
```
Comment (richardesigns) [inline faq/index.md:40]:
```suggestion
To see all the key references stored, select the storage tab in web developer tools.
```

File: faq/index.md [hunk 4]
```diff
@@ -16,37 +16,57 @@ You can always fall back to using email authentication to login. If your device
...
 
 Some users are also aware of the privacy concerns around the business models of Facebook and Google and their motivations for providing login functionality for tracking purposes.  
 
-DID offers a simple, secure solution that is easier to use than social logins and respects the user’s privacy.
+DID.app offers a simple, secure solution that is easier to use than social logins and respects the user’s privacy.
+
+### Why does DID.app send a magic link rather than a code?
+
+
+When copying a code it is possible for an attacker to mislead a user in to copying a code into the attackers website.
```
Comment (richardesigns) [inline faq/index.md:65]:
```suggestion
When copying a code it is possible for an attacker to mislead a user in to copying a code into the attacker's website.
```

File: faq/index.md [hunk 5]
```diff
@@ -16,37 +16,57 @@ You can always fall back to using email authentication to login. If your device
...
+This enables the attacker to compromise the users account.
```
Comment (richardesigns) [inline faq/index.md:66]:
```suggestion
This would enable the attacker to compromise the user's account.
```

File: faq/index.md [hunk 6]
```diff
@@ -16,37 +16,57 @@ You can always fall back to using email authentication to login. If your device
...
+
+Requiring users to click on a link is more secure than asking them to copy a code.
+The link includes the correct domain and is more resistant to phishing attack described above.
```
Comment (richardesigns) [inline faq/index.md:69]:
```suggestion
The link includes the correct domain and is more resistant to kind of phishing attack described above.
```

Comment (CrowdHailer) [reply faq/index.md:69]: @richardesigns this doesn't quite make sense to me

Comment (richardesigns) [reply faq/index.md:69]: @CrowdHailer You're right, I missed the 'the'.

Comment (CrowdHailer) [reply faq/index.md:69]: Ah yes, thanks.

**Pull request merged**